sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -155,4 +229,13 @@ public HttpPipeline getHttpPipeline() {
             () -> protocolLayer.getDigitalTwins().listRelationshipsSinglePageAsync(digitalTwinId, relationshipName),
             nextLink -> protocolLayer.getDigitalTwins().listRelationshipsNextSinglePageAsync(nextLink));
     }
+
+    private static boolean isJSONValid(String jsonInString ) {
```

Comment:
https://azure.github.io/azure-sdk/java_design.html#naming-patterns

Should cased be `isJsonValid`, although I'd recommend rearranging the words to `isValidJson`.